bindata/network/multus/002-rbac.yaml

```diff
@@ -63,3 +63,48 @@ subjects:
 - kind: ServiceAccount
   name: multus
   namespace: openshift-multus
+
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  labels: 
+    rbac.authorization.k8s.io/aggregate-to-view: "true"
+  name: multus:view
+rules:
+- apiGroups: ["apiextensions.k8s.io"]
+  resources:
+  - customresourcedefinitions
+  - customresourcedefinitions/status
```

Comment:
A multus-specific policy should not be giving people permission to view all CustomResourceDefinition objects. If that's actually needed, it should be added to the base role bindings in origin.
(Also, including the `/status` subresource explicitly should not be needed if you're not specifying different permissions from the parent resource.)

But I think this isn't needed anyway; you should be able to give people permission to view net-attach-defs without them also needing permission to view CRDs. I think.